ng2-components/ng2-activiti-analytics/src/components/widgets/number/number.widget.html

```diff
@@ -1,4 +1,4 @@
-<div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label number-widget" [formGroup]="formGroup">
+<div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label adf-number-widget" [formGroup]="formGroup">
```

Comment:
The "mdl-textfield mdl-js-textfield mdl-textfield--floating-label" are part of the MDL library